src/content/changes/highlights/2024-10-10.md

```diff
@@ -0,0 +1,66 @@
+---
+date: 2024-10-10T14:00:00
+title: Highlights for the week ending 2024-10-10
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## Observability
+
+- [alloy-gateway-app](https://github.com/giantswarm/alloy-gateway-app) version [0.1.0](https://github.com/giantswarm/alloy-gateway-app/releases/tag/v0.1.0)
+    - Create initial app version.
+
+- [alloy-app](https://github.com/giantswarm/alloy-app) version [0.6.1](https://github.com/giantswarm/alloy-app/compare/v0.5.2...v0.6.1)
+    - Bump Chart appVersion to v1.4.2.
+    - Fix circleci config.
+    - Add PodLogs as helm chart template.
+    - Upgrade Alloy upstream chart from 0.7.0 to 0.9.1.
+    - This bumps the version of Alloy from 1.3.1 to 1.4.2.
+    - Some debug metrics for components have changed.
+    - Helm chart changes, see [Alloy Helm chart v0.9.0 CHANGELOG](https://github.com/grafana/alloy/blob/helm-chart/0.9.0/operations/helm/charts/alloy/CHANGELOG.md)
+    - Fix CiliumNetworkPolicy to allow outgoing traffic to other nodes when running Alloy in clustering mode
+
+- [grafana-app](https://github.com/giantswarm/grafana-app) version [2.16.3](https://github.com/giantswarm/grafana-app/compare/v2.16.0...v2.16.3)
+    - Fix CI builds on tags.
```

Comment:
```suggestion
    - Upgrade grafana from 11.1.3 to 11.2.1.
    - Upgrade grafana helm chart from 8.3.4 to 8.5.2.
```

It looks weird to me that our changelog entries do not have the same format :(

Reply:
yeah this is something to improve